components/google_calendar/actions/free-busy-calendars/free-busy-calendars.js

```diff
@@ -0,0 +1,16 @@
+const googleCalendar = require("../../google_calendar.app");
+
+module.exports = {
+  key: "google_calendar_free_busy_calendar_info",
+  name: "Retreive Free/Busy Calendar Details",
+  description: "Retreive Free/Busy Calendar Details from the user account.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleCalendar,
+  },
+  async run() {
+    const calendar = this.googleCalendar.calendar();
+    return (await calendar.freebusy.freebusy()).data;
```

Comment:
I would recommend creating a new function named `queryFreeBusy` in `google_calendar.app.js` like this 
```javascript
    async queryFreeBusy(params) {
      const calendar = this.calendar();
      const { data } = await calendar.freebusy.freebusy(params);
      return data;
    },
```
following the same pattern as the other functions. Then you can call the function like this
```javascript
    return this.googleCalendar.queryFreeBusy();
```